### ISSUE TITLE
jkim3211

### ISSUE THREAD

**Issue by jkim3211** [RoboJackets/jr#9]
# Week 7: Git Challenge

Welcome to week 7 of software training! This week, you'll be completing a series of `git` and GitHub challenges. **To get started, type your GT username into the Title box above, then click the "Submit new issue" at the bottom of this page!** From there, work through the rest of this checklist.

If you need any help, we recommend these online resources:
- https://help.github.com/articles/github-glossary/
- https://try.github.io/levels/1/challenges/1
- https://learngitbranching.js.org/
- https://chris.beams.io/posts/git-commit/

You can, of course, also flag down an instructor if you run into anything.

## Your objectives

Replace `gburdell3` with your GT username anywhere it appears in this list.

- [x] Star this repository.
- [x] Create a fork of this repository.
- [x] Clone your fork to your computer.
- [x] Set your forked repository as a remote named `origin`.
- [x] Set our repository as a remote named `rj`.
- [x] Create a branch named your GT username (ex. `gburdell3`).
- [x] Use the terminal to create a folder named `2017` in the root of the repo.
- [x] Use the terminal to create an empty text file named `gburdell3.txt` in the `2017` folder.
- [x] Stage and commit the file (make sure you leave good commit messages!)
- [x] Create a new branch from master named `gburdell3/add-text`.
- [x] Add some ASCII art to your file (no longer than 30 lines, no more than 80 characters per line) on that branch and merge it into the `gburdell3` branch.
- [x] Make a merge conflict happen between `gburdell3` and a branch named `gburdell3/merge-conflict`.
- [x] Resolve the merge conflict.
- [ ] Create two branches named `b1` and `b2`.
- [ ] Commit a file called `stuff1.txt` in `b1`.
- [ ] Commit a file called `stuff2.txt` in `b2`.
- [ ] Take the changes from `b1` and apply them to `b2`.
- [ ] In `b2`, delete both files and commit the changes.
- [ ] Take the changes from `b2` and apply them to `b1`.
- [ ] Take the changes from `b1` and apply them to the `gburdell3/add-text` branch (this should not cause a merge conflict).
- [ ] Delete all local branches except for `master` and `gburdell3`.
- [ ] Send your committed changes to the `gburdell3` branch on your fork
- [ ] Create a pull request from your fork’s `gburdell3` branch to our master branch and ask one of the instructors in your session to approve it (if you try to merge it you won’t be able to because permissions). **In your pull request description, add a line that says "Fixes #xx" where #xx is the number of the issue you created.** This will auto-close your issue once your pull request is merged.
